middlewares/metrics.go

```diff
@@ -1,12 +1,35 @@
 package middlewares
 
 import (
+	"fmt"
 	"net/http"
+	"os"
 	"time"
 
+	"github.com/DataDog/datadog-go/statsd"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+var statsdClient *statsd.Client
+
+func initStatsdClient() {
+	statsdHost := os.Getenv("STATSD_HOST")
+	statsdPort := os.Getenv("STATSD_PORT")
+	applicationNamespace := os.Getenv("STATSD_APP_NAME")
+
+	var err error
+	statsdClient, err = statsd.New(statsdHost + ":" + statsdPort)
+	if err != nil {
+		fmt.Println("** We were unable to get a statsd client.")
+	}
+
+	statsdClient.Namespace = applicationNamespace
+}
+
+func getStatsdClient() *statsd.Client {
```

Comment:
Minor: golang recommends not calling getters "get<something>", but rather just calling it "<something>". So this would be called `statsdClient()`, or even just `client()` if you were to move this to its own namespace called "statsd"